README.md

```diff
@@ -41,9 +41,34 @@ are package private and are not part of the public API.
   * [AccessExpression](src/main/java/org/apache/accumulo/access/AccessExpression.java).
   * [Authorizations](src/main/java/org/apache/accumulo/access/Authorizations.java).
 
+## Getting Started
+
+Add the library to your CLASSPATH. For Maven, use:
+
+```xml
+<dependencies>
+  <dependency>
+    <groupId>org.apache.accumulo</groupId>
+    <artifactId>accumulo-access</artifactId>
+    <version>$version</version>
+  </dependency>
+</dependencies>
+```
+
+## Running the Example
```

Comment:
It would be nice to link to the example in the readme so its easy to jump to when browsing on GH.  Not sure how linking in a markdown header looks though.

```suggestion
## Running the [Example](src/test/java/example/AccessExample.java)
```

Reply:
Can try it